pimd/pim_bsm.c

```diff
@@ -1315,8 +1307,6 @@ int pim_bsm_process(struct interface *ifp, pim_sgaddr *sg, uint8_t *buf,
 	}
 
 	bshdr = (struct bsm_hdr *)(buf + PIM_MSG_HEADER_LEN);
-	pim_inet4_dump("<bsr?>", bshdr->bsr_addr.addr, bsr_str,
-		       sizeof(bsr_str));
 	if (bshdr->hm_len > IPV4_MAX_BITLEN) {
```

Comment:
`IPV4_MAX_BITLEN` and log message below must be fixed

Reply:
Done